brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3384 Improvement : Popup builder - Added an filter to modify query args.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3384

### Screenshots

Screencast - https://d.pr/v/aEjgYG
```
function filter_popup_query( $query_args ) {
    // Modify $query_args values.
    // Ex.
    $query_args['posts_per_page'] = 11;
    return $query_args;
} 
add_filter( 'uagb_popup_query_args', 'filter_popup_query', 10, 1 );
```

### Types of changes




- Improvement to fix bug

### How has this been tested?

- Create a 11 popup on Custom Element
- In master if you select the first one to display on any block click it will not visible on frontend
- Now switch to the branch, Add a filter in function.php of theme increase the limit of post per page in query args.
- Now add a first popup on any block click by adding custom event class in Advance panel > CSS classes and visit frontend.
- Click on that block make sure first popup will be visible.

### Checklist:
